index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto&display=swap" rel="stylesheet">
+    <title>My First Portfolio</title>
+</head>
+<body>
+    <nav class="mainMenu">
+        <p class ="menuLogo">Ruri Enciso</p>
+        <div class="menuIcon">
+            <div class="menuLvl"></div>
+            <div class="menuLvl"></div>
+            <div class="menuLvl"></div>
+        </div>
```

Comment:
- [ ] Dear @RuriEnciso, Pleas wrap your hamburger button with a nav tag, Remember your navigation items are to go inside here.  The `nav` tag is for semantics reasons.